Makefile

```diff
@@ -138,7 +145,7 @@ $(OUTPUT).elf	:	$(OFILES)
 
 #---------------------------------------------------------------------------------
 %.bin: %.elf
-	@$(OBJCOPY) -O binary $< $@
+	@$(OBJCOPY) --set-section-flags .bss=alloc,load,contents -O binary $< $@
```

Comment:
Nevermind what @b1l1s said above, as he didn't read the question right.
What that line does, is it appends the bss section to the binary.
It shouldn't have any effect on the Gateway loader, nor bootstrap.